Swift_basics_hometask_02.playground/Contents.swift

```diff
@@ -90,28 +235,104 @@ import UIKit
 // Реализуйте программу, которая подсчитает полную стоимость обучения всей группы с помощью конструкции if-else.
 // Программа должна получать на вход количество студентов, которые собираются обучаться.
 
-// MARK: - Task 2
+let gues = 10
 
+func getPrice(studentsCount: Int) -> Int {
+    switch studentsCount {
+    case 0..<5:
+        return 1000
+    case 5..<9:
+        return 850
+    case 9..<13:
+        return 650
+    case 13...:
+        return 500
+    default:
+        return 0
+    }
+}
+
+getPrice(studentsCount: 14)
+
+// MARK: - Task 2
+print("Task 2 ...")
 // Используя массивы, сформируйте группу студентов, готовую посещать курсы.
 // Для каждого студента добавьте Фамилию, возраст и флаг, который будет хранить информацию, есть ли у него/нее в наличии техника Apple.
 // Отсортируйте группу: а) по возрасту, б) по Фамилиям
 
+var students = [(name: "Ilya", age: 21, isHasMac: true),
+                (name: "Katya", age: 22, isHasMac: true),
+                (name: "Fedora", age: 23, isHasMac: false),
+                (name: "Susanna", age: 20, isHasMac: false),
+                (name: "Yan", age: 20, isHasMac: true),
+                (name: "Yan", age: 19, isHasMac: true),
+                (name: "Fedora", age: 13, isHasMac: false),
+                (name: "Ilya", age: 19, isHasMac: true)
+]
+
+students.sort { $0.age < $1.age }
+students
+
+students.sort { $0.name < $1.name }
+students
+
 // MARK: - Task 3
+print("Task 3 ...")
 // Добавьте возможность случайным образом решить, какой студент пойдет у доске отвечать на вопросы преподавателя
 
-// MARK: - Task 4
+let randomStudent = students.randomElement()
 
+// MARK: - Task 4
+print("Task 4 ...")
 // Реализуйте программу, которая подсчитает полную стоимость обучения всей группы,
 // с учетом того, что обучаться могут только те студенты, у которых есть в наличии техника Apple.
 // Используете решения из предыдущих задач
 
+var studentCount: Int = 0
+
+for student in students {
```

Comment:
if it is different ways how u can do it add comments and calculate price for each method what u use